Docsite template: make it ready-to-use, part 1 (#8)
ansible-community/project-template: docs/contributor_guide.md

# Contributor guide

Welcome to the PROJECTNAME Contributor guide!

> > If your project is not going to have a docsite (in case it is very simple and having a docsite would be an overkill), the following information should be present in your project CONTRIBUTING.md file and referred to from the [README template](https://github.com/ansible-community/project-template/blob/main/README.md).

Reviewer: ```suggestion
> If your project is not going to have a docsite (in case it is very simple and a docsite would be overkill), the following information should be present in your project CONTRIBUTING.md file and referred to from the [README template](https://github.com/ansible-community/project-template/blob/main/README.md).
```

> > In the following sections, if the content is as simple as a few lines, put it in there, otherwise create separate files and refer to them from the sections. Add other sections if needed.

Reviewer: ```suggestion
> In the following sections, if the content is as simple as a few lines, include it in the section. Otherwise create separate files and refer to them from the sections. Add other sections if needed.
```

Need help or want to share feedback? See the [Community guide](community_guide.md)!

## Project place in the Ansible ecosystem

> Describe a project place in the Ansible ecosystem here. For example, the project can be a component of another project or serve as an auxiliary project.
In case the project is a standalone self-sufficient tool, this section can be omitted.

## Non-code contributions

### Ask and answer questions

Ask and answer questions. Reach out and communicate with your fellow users and contributors.
See the official communication channels in the [Community guide](community_guide.md).

### Contributing to documentation

Reviewing, fixing, and maintaining the documentation is a great way to contribute to PROJECTNAME. 

> Describe how to contribute to the documentation or list references to separate documentation guidelines. Put a GitHub link to issues with documentation-related labels.

### How to open an issue

If you want to report a bug or request a new feature:

1. Search in the [issues](https://github.com/ORG/REPO/issues) for similar reports/requests.
2. If there are no such issues yet, open a new one by clicking the `New issue` button.

## Code contributions

> > Consider having a separate code contributions guide and a link to it from here.

Reviewer: ```suggestion
> Consider having a separate code contributor guide and a link to it from here.
```

### Project architecture

> Describe your project architecture here: its components and how they work together.

### Getting started

> > Create the `easyfix` label in the repository and mark issues that are easy to fix for newcomers.

Reviewer 2: The github default is 'good first issue' so we should recommend that here instead of easyfix. The filters we share to find easy issues use both labels.

Author: fixed

> If you have no specific idea for contribution, please take a look at the [easyfix issues](https://github.com/ORG/REPO/labels/easyfix).

Reviewer 2: same comment on 'easy first issue'

> Create a simple and consistent getting started development guide that includes all a person needs to know to make their first contribution from forking the repo and setting up a development environment to running tests and submitting a pull request. See the [Creating your first collection pull request](https://docs.ansible.com/ansible/devel/community/create_pr_quick_start.html) guide as an example.

### Testing

> Describe how the testing works in the project, how to add all kinds of tests and run them locally.

### Reviewing open issues and pull requests

> Reviewing open issues and pull requests is a great way to contribute to PROJECTNAME. Even if you have no coding experience, you can share your opinion on requested features/their implementations from user's perspective.

Reviewer 2: ```suggestion
Reviewing open issues and pull requests is a great way to contribute to PROJECTNAME. Even if you have no coding experience, you can share your opinion on requested features/their implementations from a user's perspective.
```
